ddtrace/tracer/option.go

```diff
@@ -1065,8 +1065,8 @@ func WithStatsComputation(enabled bool) StartOption {
 // This option is only intended to be used by Orchestrion https://github.com/DataDog/orchestrion
 func WithOrchestrion(metadata map[string]string) StartOption {
```

Comment:
@ahmed-mez Thought: should we add this option automatically when instrumenting with Orchestrion?

I assume this should be done when [initializing the tracer through `instrument.Init`](https://github.com/DataDog/orchestrion/blob/add25c8081a1eb17c98ed10577d0196c39c3314c/instrument/instrument.go#L70).

Reply:
Yes!! That will be my next PR in Orchestrion, just waiting for the v1.55.0 rc1 tag of dd-trace-go to be published to import it in Orchestrion!